_security-plugin/index.md

```diff
@@ -8,20 +8,66 @@ redirect_from:
+
+To learn more about this feature, see the [Access control]({{site.url}}{{site.baseurl}}/security-plugin/access-control/index/) section of security documentation.
+
+### Audit logging and compliance
+
+Finally, audit logging and compliance refer to mechanisms that allow for tracking and analysis of activity within the cluster. This is important after events in which sensitive data is breached (unauthorized access) or suffers unintended exposure, as when data is sent to an unsecure location and left vulnerable. However, audit logging can be just as valuable a tool for assessing excessive loads on the cluster or surveying trends for a given task. This feature allows you to oversee changes made anywhere in the cluster and track access patterns and API requests of all types, whether valid or invalid.
+
+How OpenSearch archives logging is configurable at many levels of detail, and there are a number of options for where those logs are stored. Compliance features also ensure that all data is available if and when compliance auditing is required. In this case, the logging can be automated to focus on data especially pertinent to those compliance requirements.
+
+See the [Audit logs]({{site.url}}{{site.baseurl}}/security-plugin/audit-logs/index/) section of security documentation to read more about this feature.
+
+## Other features and functionality
+
+Security includes other features that make working in OpenSearch easier.
+
+### Dashboards multi-tenancy
+
+One such feature is OpenSearch Dashboards multi-tenancy. Tenants are work spaces that include visualizations, index patterns, and other Dashboards objects, which can be shared with other Dashboards users. OpenSearch leverages roles to manage accesss to tenants and safely make them available to other Dashboards users.
+For more information on creating tenants, see [OpenSearch Dashboards multi-tenancy]({{site.url}}{{site.baseurl}}/security-plugin/access-control/multi-tenancy/).
+
+### Cross-cluster search
+
+Another notable feature is cross-cluster search. This feature provides a user with the ability to perform searches from one node in a cluster across other clusters that have been set up to coordinate this type of search. As with other features, cross-cluster search is supported by the OpenSearch access control infrastructure, which defines the permissions users have for working with this feature.
+To learn more, see [Cross-cluster search]({{site.url}}{{site.baseurl}}/security-plugin/access-control/cross-cluster-search/).
+
+## Next steps
+
+To get started with security, see the configuration overview in the [Security configuration]({{site.url}}{{site.baseurl}}/security-plugin/configuration/index/) section, which provides the basic steps for setting up security in your OpenSearch implementation, including links to information about customizing security for your business needs.
```

Comment:
We need a word after the first instance of "security". How about "management" or just "configuration"? Replace "implementation, including" with "implementation and includes".

Reply:
How about simply: "To get started, see..."?